test/podFailover/workload/scenarios/azDiskV2-Replicas.yaml

```diff
@@ -10,6 +10,8 @@ storageClass:
     allowVolumeExpansion: true
   provisioner: disk.csi.azure.com
 
+schedulerName: csi-azuredisk-scheduler-extender
```

Comment:
This default will only work with Standalone or SxS deployments of the driver. When using the V2 driver included in AKS, the scheduler extender is part of the default scheduling configuration and a scheduler name should not be provided.

Reply:
yes, this is intended to be added to fix the pipelines which use helm to install V2. are you saying that if schedulerName is added to an AKS installed AzDisk it will break the pod?

Reply:
The pod won't be scheduled because the requested scheduler profile does not exist.

Reply:
ok. will make a note of that in the yaml example